packages/playwright/src/runner/runner.ts

```diff
@@ -161,7 +162,12 @@ async function writeLastRunInfo(testRun: TestRun, status: FullResult['status'])
   await fs.promises.mkdir(outputDir, { recursive: true });
   const lastRunReportFile = path.join(outputDir, '.last-run.json');
   const failedTests = testRun.rootSuite?.allTests().filter(t => !t.ok()).map(t => t.id);
-  const lastRunReport = JSON.stringify({ status, failedTests }, undefined, 2);
+  const testDurations = testRun.rootSuite?.allTests().reduce((map, t) => {
+    if (t.results.length)
+      map[t.id] = t.results.reduce((a, b) => a + b.duration, 0);
+    return map;
+  }, {} as { [testId: string]: number });
```

Comment:
I'm actually not sure it is the right way to sum all the durations… maybe it makes more sense to calc the average? Or only include durations from successful test runs… 🤔

Reply:
I think this is ok for now, we'll iterate based on feedback.